spec/helpers/help_page_helper_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-describe HelpPageHelper do
+describe HelpPageHelper, type: :helper do
```

Comment:
my rubocop thinks this isn't neede as inferred - https://docs.rubocop.org/rubocop-rspec_rails/cops_rspecrails.html#rspecrailsinferredspectype